projects/cashmere/src/lib/datepicker/datepicker.component.ts

```diff
@@ -66,6 +66,19 @@ export class HcDatepickerContentBase {
 export class DatepickerComponent implements OnDestroy {
     private _scrollStrategy: () => ScrollStrategy;
 
+    /**
+     * Whether the datepicker can be set to a blank value (by deleting its contents). Defaults to `false`.
+     */
+    @Input('allows-blank-values')
+    get allowsBlankValues() {
+        return this._datepickerInput && this._datepickerInput.allowsBlankValues;
+    }
+    set allowsBlankValues(val) {
```

Comment:
Not sure if this matters, but should val be set to a type of boolean here? should you use parseBooleanAttribute(val) as well?

Reply:
Good Q. @andrew-frueh ? Can you help answer?

Reply:
Great catch on your review @cikeddy - yes, `parseBooleanAttribute` is the preferred way to handle a boolean input so that we can accept either string, number, or boolean values for that boolean param.  So if either of you could make that switch, then we should be good to merge this one.  Thanks to both of you!